Doc/whatsnew/3.9.rst

```diff
@@ -70,6 +70,51 @@ Summary -- Release highlights
 New Features
 ============
 
+Dictionary Merge & Update Operators
+-----------------------------------
+
+Merge (``|``) and update (``|=``) operators have been added to the built-in
+:class:`dict` class.
+
+.. XXX The rest is just a copy-paste of the PEP's Specification section.
```

Comment:
Maybe drop the rest and link to the PEP instead? The comment above praises brevity. :-)

Reply:
I think the brevity bit is about the summary section, no? I latched onto "Do not spend very much time on the wording of your changes"... hence, the copy-paste. But you're right that it probably does not add much here.